static/app/components/charts/simpleTableChart.tsx

```diff
@@ -81,12 +81,9 @@ function SimpleTableChart({
   }
 
   const meta = metadata ?? {};
-  const usingEvents = organization.features.includes(
-    'discover-frontend-use-events-endpoint'
-  );
   const columns = decodeColumnOrder(
     fields.map((field, index) => ({field, alias: fieldAliases[index]})),
-    usingEvents
+    true
```

Comment:
We spoke about this, but to clarify you plan to remove this argument in another PR because we're going to force it to be `true`? I guess there are a lot of functions that could be cleaned up and this PR is big already

Reply:
Yup this will be handled in a follow up pr! Specifically the issue is that `decodeColumnOrder` is still being called in multiple places, sometimes with the second arg intentionally being defaulted to `false/undefined`. Because of this, it won't be as simple as blanket removing this arg so we'll have to do further clean up here.